examples/3d/scattering3d_qm7.py

```diff
@@ -1,3 +1,9 @@
+"""
+3D scattering quantum chemestry regression
```

Comment:
chemestry -> chemistry

Reply:
This is not the purpose of this PR.

Reply:
I see it in the diff. So if it's an addition to the codebase, it should be made correct in this PR. If it's already on the master branch, you are right that we should fix the spelling in a separate PR, but I don't think it is the case.